src/cladetime/tree.py

```diff
@@ -125,14 +148,6 @@ def _get_tree_url(self):
         )
         return tree_url
 
-    def _get_url_ncov_metadata(self):
```

Comment:
No longer needed because we're setting the `Tree` ncov metadata information the same way we set the `CladeTime` ncov metadata property